backend/src/api/routes/chat.py

```diff
@@ -135,7 +135,7 @@ async def chat(
     # create_session = await session_repo.read_create_sessions_by_id(id=chat_in_msg.sessionId, account_id=chat_in_msg.accountID, name=chat_in_msg.message[:20])
     # response_msg = await rag_chat_repo.get_response(session_id=session_id, input_msg=chat_in_msg.message, chat_repo=chat_repo)
 
-    # TODO: name=chat_in_msg.message[:20] can be edited
+    # TODO: name=chat_in_msg.message[:20] use to create uuid in here, we use username to create session in /api/seesionuuid. Is that acceptable? @Micost
```

Comment:
@Micost Today we add a new api `chat/sessionuuid` get new session. And it create sesson with current user name because it doesn't have input(we call it before conversation starting). So, here we create  session with `chat_in_msg.message[:20]`.  PS: we check the code here, it will never create new session. So, here may need you help to check the code inside `session_repo.read_create_sessions_by_uuid`.